dashboard/cron.py

```diff
@@ -42,221 +41,60 @@
-                   " from assignment_fact af inner join assignment_dim ad on af.assignment_id = ad.id where af.course_id='" + UDW_COURSE_ID + "'" \
-                   "and ad.visibility = 'everyone' and ad.workflow_state='published')" \
-                   "select * from assignment_info"
-
-    return HttpResponse("loaded assignment info: " + util_function(assignment_sql,'assignment'))
-
-
-def submission(request):
-    '''
-    student submission information for assignments
-    :param request:
-    :return:
-    '''
-    logger.info("update_submission(): ")
-    submission_url = "with sub_fact as (select submission_id, assignment_id,user_id, global_canvas_id, published_score " \
-                     "from submission_fact sf join user_dim u on sf.user_id = u.id where course_id = '" + UDW_COURSE_ID + "')," \
-                     "enrollment as (select  distinct(user_id) from enrollment_dim where course_id = '" + UDW_COURSE_ID + "' and workflow_state='active' " \
-                     "and type = 'StudentEnrollment'),"\
-                     "submission_time as (select id, graded_at, graded_at at time zone 'utc' at time zone 'America/New_York' as local_graded_time from submission_dim),"\
-                     "sub_with_enroll as (select sf.* from sub_fact sf join enrollment e on e.user_id = sf.user_id),"\
-                     "submission as (select se.submission_id,se.assignment_id,se.global_canvas_id,se.published_score, st.graded_at, st.local_graded_time " \
-                     "from sub_with_enroll se inner join submission_time st on se.submission_id = st.id)" \
-                     "select submission_id AS ID, assignment_id AS assignment_id, global_canvas_id AS user_id, " \
-                     "published_score AS score, graded_at AS graded_date, local_graded_time as local_graded_date from submission"
-
-    return HttpResponse("loaded assignment submission info: " + util_function(submission_url,'submission'))
-
-
-def weight_consideration(request):
-    '''
-    load the assignment weight consider information with in a course. Some assignments don't have weight consideration
-    the result of it return boolean indicating weight is considered in table calculation or not
-    :param request:
-    :return:
-    '''
-    logger.info("weight_consideration()")
-    is_weight_considered_url ="with course as (select course_id, sum(group_weight) as group_weight from assignment_group_fact " \
-                              "where course_id = '" + UDW_COURSE_ID + "' group by course_id having sum(group_weight)>1)"\
-                              "(select CASE WHEN EXISTS (SELECT * FROM course WHERE group_weight > 1) THEN CAST(1 AS BOOLEAN) ELSE CAST(0 AS BOOLEAN) END)"
-
-    return HttpResponse("loaded weight_consideration info: " + util_function(is_weight_considered_url,'assignment_weight_consideration', 'weight'))
+UDWCourseIdsString = config("CANVAS_COURSE_IDS", "")
+UDWCourseIds = UDWCourseIdsString.split(",")
```

Comment:
Rather than having this split, just change the line before to be 

config("CANVAS_COURSE_IDS", default="", cast=Csv()) 

This is already imported.